sms_wsj/database/wsj/write_wav.py

```diff
@@ -64,7 +110,7 @@ def resample_with_sox(x, rate_in, rate_out):
         stderr=subprocess.PIPE,
         input=x.tobytes(order="f")
     )
-    return np.fromstring(process.stdout, dtype=np.float32)
+    return np.frombuffer(process.stdout, dtype=np.float32)
```

Comment:
couldnt we just switch to the resampler in paderbox?

Reply:
The function in `paderbox` improves the stability of sox for arbitrary input.
Since the output from it is slightly different, I like to stick with this function for reproducibility.
(Here, we don't hit the stability problem)

Reply:
okay, sure in that case I have already approved this pull request :grin: